runtime/runtime/src/lib.rs

```diff
@@ -2649,9 +2650,9 @@ impl<'a> ApplyProcessingState<'a> {
     }
 }
 
-#[derive(Clone, Copy)]
-pub struct SignedValidPeriodTransactions<'a> {
-    transactions: &'a [SignedTransaction],
+#[derive(Clone)]
+pub struct SignedValidPeriodTransactions {
```

Comment:
What is the lifetime of a transaction nowadays? Is it `Signed` -> `SignedValidPeriod` -> `Validated`?

Reply:
Instead of having a `struct{ Vec<SignedTransaction>, Vec<bool> } can we have a Vec<`SignedValidPeriodTransaction`>?